docs/setup/setup.md

```diff
@@ -58,58 +58,57 @@ docker pull ghcr.io/project-monai/monai-deploy-informatics-gateway:0.1.0
 
 ## Configure Informatics Gateway
 
-Use the following command to initialize Informatics Gateway & default configuration:
+Use the following commands to initialize the Informatics Gateway and default configuration:
 
 ```bash
 mig-cli config init
 mig-cli config endpoint http://localhost:5000
 ```
 
-
 ```powershell
 mig-cli.exe config init
 mig-cli.exe config endpoint http://localhost:5000
 ```
 
-This first command extract the default `appsettings.json` file into the home directory:
+The first command extracts the default `appsettings.json` file into the home directory:
 
 * Linux: `~/.mig/appsettings.json`
 * Windows: `C:\Users\[username]\.mig\appsettings.json`
 
 This file is mapped into the Docker container and used by the Informatics Gateway when `mig-cli` is used to launch the application.
 
-For a complete reference of the `appsettings.jon`, please refer to [Configuration Schema](schema.md).
+For a complete reference of the `appsettings.json` file, refer to the [Configuration Schema](schema.md).
 
-The second command tells `mig-cli` where the endpoint is for the Informatics Gateway RESTful API.
+The second command passes the endpoint for the Informatics Gateway RESTful API to `mig-cli`.
 
 > [!Note]
-> To see available commands simply execute `mig-cli` or `mig-cli.exe`.
+> To see available commands, simply execute `mig-cli` or `mig-cli.exe`.
 
 ## Storage Consideration & Configuration
 
-Informatics Gateway operates on two storage locations. The first is for temporarily storing of the incoming data for data grouping. The second is where Informatics Gateway uploads grouped datasets to final storage shared by other MONAI Deploy sub-systems.
+The Informatics Gateway operates on two storage locations. In the first location, the incoming data for data grouping is temporarily stored. In the second location, the Informatics Gateway uploads grouped datasets to final storage shared by other MONAI Deploy sub-systems.
 
 ### Temporary Storage of Incoming Dataset
 
-The temporary storage location, by default, is set to `/payloads` in the `appsettings.json` file.
+By default, the temporary storage location is set to `/payloads` in the `appsettings.json` file.
 
-To change the temporary storage location, please locate `./InformaticsGateway/storage/temporary` property in the `appsettings.json` file.
+To change the temporary storage location, locate the `./InformaticsGateway/storage/temporary` property in the `appsettings.json` file and modify it.
 
 > [!Note]
-> Calculate the required temporary storage based on the number of studies and the size of each study.
-> Please also consider the AE Title timeout if the AE Title needs to wait a long time before assembling & uploading
+> You will need to calculate the required temporary storage based on the number of studies and the size of each study.
+> Also, consider changing the AE Title timeout if the AE Title needs to wait a long time before assembling and uploading
```

Comment:
It's unclear in the language whether the reader needs to consider changing the value of the AE Title timeout or 
or enabling it altogether (i.e. is it enabled by default?). I assumed the former when rewriting.

Reply:
Users shall consider changing the default timeout of 5 seconds depending on how they are sending the DICOM dataset.

Reply:
Got it. No changes required then.